kubelet/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - kubelet
 
+## 1.4.0 / Unreleased
+
+* [CLEANUP] Handle exceptions inside `retrieve_pod_list`, instead of outside the function. See [#1648][]
+
```

Comment:
I don't think the CHANGELOG needs typo be manually updated in this repo anymore. I think it is done automatically using the PR title and the PR labels. Explained in this doc https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md.

Reply:
OK, thanks for the info!

Reply:
My bad, I should've mentioned that in my comment, a lot of things happening these days... thanks @devonboyer for pointing that out